src/art/art.cpp

```diff
@@ -271,15 +571,20 @@ asf::auto_release_ptr<asr::Project> build_project(const char* file, const char*
     // Create an empty project.
     asf::auto_release_ptr<asr::Project> project(asr::ProjectFactory::create("test project"));
     project->search_paths().push_back_explicit_path("build/Debug");
+    // ***** add precompiled shaders path
+    // TODO: make this dynamic to the project
+    project->search_paths().push_back_explicit_path("/Users/christopher/Workspace/appleseed/shaders/appleseed");
```

Comment:
should be able to call bu_dir() here to search where appleseed is expected to be installed, or to build up some other path manually (see include/bu/dir.h for examples).